Typescript error for interface ID

### Version of react-native-iap
3.4.13

### Version of react-native
0.61.1

### Steps to reproduce the behavior
```
$ npx tsc
node_modules/react-native-iap/index.d.ts:5:22 - error TS2693: 'string' only refers to a type, but is being used as a value here.

5 interface ID extends string {}
                       ~~~~~~
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -2,7 +2,7 @@ import { EmitterSubscription } from 'react-native';
 
 import * as Apple from './apple';
 
-interface ID extends string {}
+interface ID extends String {}
 
 interface Common {
   title: string;

